gdpr_assist/models.py

```diff
@@ -246,33 +278,278 @@ def _cast_class(cls, model, privacy_meta):
 
         return model
 
+    def get_anon_log(self):
+        """
+        See if there are any related models which need to be anonymised.
+
+        They will be any reverse relations to PrivacyModel subclasses where their
+        OneToOneField and ForeignKey on_delete is ANONYMISE.
+        """
+        if not self.anonymised:
+            return "%s is not anonymised yet." % self
+
+        from gdpr_assist.models import EventLog
+        logs = EventLog.objects.order_by('log_time')
+
+        top_level_log_lines = EventLog.objects.for_instance(self).values('log_time', 'event', 'app_label', 'model_name',
+                                                                        'target_pk', 'acting_user')
+
+        if top_level_log_lines.count() > 0:
+            actual_anon_log_line_start = top_level_log_lines.filter(event=EventLog.EVENT_RECURSIVE_START).last()
+            actual_anon_log_line_end = top_level_log_lines.filter(event=EventLog.EVENT_ANONYMISE).last()
+
+            lines = logs.filter(log_time__gte=actual_anon_log_line_start["log_time"],
+                                log_time__lte=actual_anon_log_line_end["log_time"]). \
+                values('log_time', 'event', 'app_label', 'model_name', 'target_pk', 'acting_user')
+
+            user = str(actual_anon_log_line_start["acting_user"]) if actual_anon_log_line_start[
+                                                                   "acting_user"] is not None else "[Non-descript user]"
```

Comment:
Because `acting_user` is a FK to the `User` model and I don't want to manually grab the first name, last name, and email from that object. 

What do you have against `str`?

And btw, there's no button to reject your suggested change, but I reject it, because it would cause an error (source: tried it previously)

Reply:
```bash
In [7]: EventLog.objects.last()

In [8]: e = _

In [9]: e.user
Out[9]: <django.db.models.fields.related_descriptors.RelatedManager at 0x115e40e10>

In [10]: str(e.user)
Out[10]: 'seminar.ExcusedPenalty.None'

In [11]: print "%s" % e.user
seminar.ExcusedPenalty.None

In [12]: print "%s" % str(e.user)
seminar.ExcusedPenalty.None
```